pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.black]
+target-version = ["py39"]
+
+[tool.ruff]
+select = ["E", "F", "C4", "I001", "UP", "SIM"]
+ignore = ["E501", "SIM105"]
+target-version = "py39"
+extend-exclude = ["vendor"]
+
+[tool.ruff.pyupgrade]
+keep-runtime-typing = true
```

Comment:
Does this fix that issue where Talon choked on the newer type annotations?

Reply:
I _think_ so, but not 100% certain. Here's the docs for this setting: https://beta.ruff.rs/docs/settings/#keep-runtime-typing

Reply:
Huh. Strange that it says:

> Note that this setting is only applicable when the target Python version is below 3.9 and 3.10 respectively

So I'm not sure this rule even applies? Maybe instead we should just disable `UP006`? That corresponds to [PEP 585](https://peps.python.org/pep-0585/), which I'm pretty sure is the troublemaker.

Reply:
Huh. Strange that it says:

> Note that this setting is only applicable when the target Python version is below 3.9 and 3.10 respectively

So I'm not sure this rule even applies? Maybe instead we should just disable `UP006`? That corresponds to [PEP 585](https://peps.python.org/pep-0585/), which I'm pretty sure is the troublemaker.

Reply:
Ok I did some local testing and looks like these rules would still have caused https://github.com/knausj85/knausj_talon/issues/922.  Disabling `UP035` did the trick, though.  I pushed updated config; lmk if it looks ok and we can merge away

Thanks for this one!  So much faster.  Now prettier is the slow horse 😄